mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -749,8 +749,16 @@ def hydra_entry_point(cfg: DictConfig) -> None:
     # Allow dynamic attributes.
     OmegaConf.set_struct(cfg, False)
 
-    # Allow dynamic attributes.
-    OmegaConf.set_struct(cfg, False)
+    # Load and merge the system and network config files with the original one
+    system_config_path = f"{hydra.utils.get_original_cwd()}/mava/configs/system/ff_ippo.yaml"
+    system_config = OmegaConf.load(system_config_path)
+    cfg.system = OmegaConf.merge(cfg.system, system_config)
+
+    network_config_path = (
+        f"{hydra.utils.get_original_cwd()}/mava/configs/network/{cfg.system.network_name}.yaml"
+    )
+    network_config = OmegaConf.load(network_config_path)
+    cfg["network"] = network_config
```

Comment:
I'm not a fan of this I think it's too much friction if we want to add configs in the future or if a name gets changed, but I do think the goal of this PR is good and we can do a lot of it with normal hydra.

Reply:
This code has been removed in the latest update.